src/Microsoft.VisualStudio.Threading/JoinableTaskCollection.cs

```diff
@@ -86,6 +75,21 @@ public string DisplayName
             set { this.displayName = value; }
         }
 
+        /// <summary>
+        /// Gets JoinableTaskContext for <see cref="JoinableTaskDependencyGraph.JoinableTaskDependentData"/> to access locks.
+        /// </summary>
+        JoinableTaskContext IJoinableTaskDependent.JoinableTaskContext => this.Context;
+
+        /// <summary>
+        /// Gets a value indicating whether we need count reference for child dependent nodes.
+        /// </summary>
+        bool IJoinableTaskDependent.NeedRefCountChildDependencies => this.refCountAddedJobs;
+
+        ref JoinableTaskDependencyGraph.JoinableTaskDependentData IJoinableTaskDependent.GetJoinableTaskDependentData()
+        {
+            return ref this.dependentData;
+        }
+
```

Comment:
nit: can we use the expression bodied member syntax here as you've used for the other two?

Reply:
ok.